Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -85,21 +87,41 @@ public final class RootTabBarViewController: UITabBarController {
       .observeForUI()
       .observeValues { [weak self] in self?.selectedIndex = $0 }
 
-    self.viewModel.outputs.scrollToTop
-      .observeForUI()
+    self.viewModel.outputs.scrollViewControllerAtIndexToTop
+      .observeForControllerAction()
+      .map { [weak self] index -> UIViewController? in
+        guard let vc = self?.viewControllers?[index] else { return nil }
```

Comment:
Again, can't remember because it's been pretty long since I made this change but I think I determined that because of this change this _should_ be impossible, but doesn't hurt to keep it in 👍